srcpkgs/lumina-pdf/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'lumina-pdf'
+pkgname=lumina-pdf
+version=1.6.0
+revision=1
+build_wrksrc="src-qt5"
+build_style=qmake
+configure_args="PREFIX=/usr L_MANDIR=/usr/share/man"
+hostmakedepends="qt5-host-tools qt5-qmake"
+makedepends="qt5-devel poppler-qt5-devel"
+short_desc="PDF reader and presentation utility from the Lumina Desktop"
+maintainer="Ken Moore <ken@project-trident.org>"
+license="BSD-3-Clause"
+homepage="https://github.com/lumina-desktop/lumina-pdf"
+distfiles="https://github.com/lumina-desktop/lumina-pdf/archive/v${version}.tar.gz"
+checksum=376f8a48b0c33bcf1603228748dd0d531eb3d59cda703a3927d950459674144b
+
+post_install() {
+	vlicense ${wrksrc}/LICENSE
```

Comment:
Since the stage is executed in wrksrc, there should be no need to define it as part of the path here.

Reply:
Unfortunately, removing the "${wrksrc}" from post_install breaks the build. I think this is due to the custom "build_wrksrc" directory getting used for all [pre/post]_install() routines as well.

Reply:
Alright then, I didn't realize that. I think we can leave it like that.